translate/filename.go

```diff
@@ -28,11 +31,20 @@ func Filename(item mapper.Item) string {
 }
 
 func SectionFilename(t mapper.Type) string {
-	dir := Dir(t.Sys.ID)
+	dir := Dir(baseContentDir, t.Sys.ID)
+
 	return dir + sectionIdxFile
 }
 
 func LeafBundleFilename(t mapper.Type) string {
-	dir := Dir(t.Sys.ID)
+	dir := Dir(baseContentDir, t.Sys.ID)
+
 	return dir + idxFile
 }
+
+// ArcheTypeFilename takes a content-type's name and returns the file path to the corresponding archetype file.
+func ArcheTypeFilename(contentTypeName string) string {
```

Comment:
Style stuff : Archetype is one word - should downcase the `T` here. Additionally I like when functions start with verbs like `Get` which would be appropriate in this case